packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -285,7 +287,7 @@ const RealAccountSignup = observer(({ history, state_index, is_trading_experienc
 
     const getModalHeight = () => {
         if (is_from_restricted_country) return '304px';
-        else if ([invalid_input_error, status_dialog].includes(getActiveModalIndex())) return 'auto';
+        else if ([invalid_input_error, status_dialog, signup_error].includes(getActiveModalIndex())) return 'auto';
```

Comment:
Hi @shahzaib-deriv 👋 
Can you please move this condition on top before if and store the value in any variable and 
 use that variable here. Like this

`const anyVariableName = [invalid_input_error, status_dialog, signup_error].includes(getActiveModalIndex())`

By doing this code will be bit more readable. 

Thank you 🙇

Reply:
Thanks for the suggestion @fasihali-deriv. Definitely will be better, however similar conditions are being used multiple times the same code block, being explicit here makes it easier to debug